### PR TITLE
wkargul/hybrid net quick launch

### PR DESCRIPTION
**Pull request by wkargul**
This PR consists of hybrid net quick launch guide. It fixes https://github.com/golemfactory/yagna/issues/2179

### CODE REVIEW

File: see-also/quick-launch-on-hybrid-net.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### Running the daemon
+
+#### Provider
+
+To run the Golem provider on hybrid net, type the following in the terminal:
+```
+golemsp run --subnet hybrid-mainnet
+```
+
+#### Requestor
```
Comment (nieznanysprawiciel) [inline see-also/quick-launch-on-hybrid-net.md:38]: We must rethink how to advertise Requestor. I don't think if they will be willing to risk mainnet funds on experimental network.
On the other side on `hybrid` devnet are no community providers, so they can't test on it.

**Review by nieznanysprawiciel — COMMENTED** [review]
You didn't mention port forwarding and potential configuration

**Comment by wkargul** [comment]
This has been already approved by reviewers 

**BUT**

remember to ensure that `YA_INSTALLER_CORE` is defined correctly before release

**Review by nieznanysprawiciel — APPROVED** [review]

File: see-also/quick-launch-on-hybrid-net.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+description: A guide to help you run the yagna daemon with the New Network Driver enabled.
+---
+
+# Quick launch on New Network Driver
+
+### Installation
+
+We have enabled providers and requestors to run their nodes with the New Network Driver.
```
Comment (shadeofblue) [inline see-also/quick-launch-on-hybrid-net.md:9]: This doesn't sound like installation instructions but rather like an introduction...

**Review by shadeofblue — COMMENTED** [review]
it's more or less okay - just I'd move the intro to before the "Installation" ... plus, maybe we should elaborate just a bit about _what_ the difference actually is ... and why it's called "hybrid"

**Review by shadeofblue — APPROVED** [review]
as discussed, :+1:

**Pull request merged**